tests/security/test_admin_attack_scenarios.py

```
@@ -458,109 +429,89 @@ def test_denial_of_service_attack_chain(self, client):
                 response = client.get(endpoint)
-                attack_chain.append(("recon", endpoint, response.status_code))
+                # Should not return sensitive information in error messages
+                if response.status_code in [500, 200]:
+                    response_text = response.text.lower()
+                    has_sensitive_info = any(
+                        word in response_text
+                        for word in ["password", "secret", "key", "token", "database", "admin_users", "session"]
+                    )
+                    attack_results.append(("recon_info_leak", endpoint, has_sensitive_info))
             except Exception:
                 pass
```

Ruff (0.12.2)

`try`-`except`-`pass` detected, consider logging the exception

(S110)